app-tracking-protection/vpn-impl/src/main/res/layout/activity_device_shield_activity.xml

```diff
@@ -139,23 +138,23 @@
                 android:paddingStart="@dimen/keyline_4"
                 android:text="@string/atp_ActivityAbout" />
 
-            <com.duckduckgo.mobile.android.ui.view.SettingsOption
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
                 android:id="@+id/cta_what_are_app_trackers"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:text="@string/atp_ActivityAppTrackersFAQ"/>
+                android:layout_height="@dimen/oneLinelistItemHeight"
```

Comment:
I see some `OneLineListItem` have `android:layout_height="@dimen/oneLinelistItemHeight"` and some have `android:layout_height="wrap_content"`. What is the rule for it?